src/lib/translations.ts

```diff
@@ -12,19 +13,37 @@ export const translations = {
     ) as typeof trackHarTranslationsDe,
 };
 
+const templateFolder = join(__dirname, '/../../templates');
```

Comment:
I know, I know. But you're not allowed to do that. Parcel is _very_ particular about when it will inline a file.